respa_admin/templates/_base.html

```diff
@@ -15,6 +18,12 @@
 </head>
 
 <body>
+    <div class="hidden" id="all-languages">
+        {% for key, value in LANGUAGES %}
+            <input value="{{ key }}" />
```

Comment:
Maybe something else than `<input>` would be better for this, since it's not really an input, but just a way to send the languages data to the Javascript code. How about using `<span>` for this.?